src/Commands/Connection/SetCommand.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Pantheon\Terminus\Commands\Connection;
+
+use Consolidation\OutputFormatters\StructuredData\RowsOfFields;
+
+use Pantheon\Terminus\Commands\TerminusCommand;
+use Terminus\Collections\Sites;
+use Terminus\Exceptions\TerminusException;
+use Terminus\Models\Environment;
+
+/**
+ * Class SetCommand
+ * @package Pantheon\Terminus\Commands\Connection
+ */
+class SetCommand extends TerminusCommand
+{
+    /**
+     * Set git or sftp connection mode on a site's dev or multidev environment
+     *
+     * @authorized
+     *
+     * @command connection:set
+     *
+     * @param string $environment Name of the environment to set. Note that you cannot change 'test' or 'live'.
+     * @param string $mode        Connection mode, one of: <git|sftp>
+     *
+     * @return bool
+     *
+     */
+    public function connectionSet($environment, $mode)
```

Comment:
@ajbarry @TeslaDethray  My point here is that you have a parameter called "$environment" that actually contains a `site.env`, not just an env. That should be renamed, don't you think?